src/category_theory/limits/shapes/wide_pullbacks.lean

```diff
@@ -191,4 +191,166 @@ abbreviation has_wide_pullbacks : Prop :=
 abbreviation has_wide_pushouts : Prop :=
 Π (J : Type v), has_colimits_of_shape (wide_pushout_shape J) C
 
+variables {C J}
+
+/-- `has_wide_pullback B objs arrows` means that `wide_cospan B objs arrows` has a limit. -/
+abbreviation has_wide_pullback (B : C) (objs : J → C)
+  (arrows : Π (j : J), objs j ⟶ B) : Prop :=
+has_limit (wide_pullback_shape.wide_cospan B objs arrows)
+
+/-- `has_wide_pushout B objs arrows` means that `wide_span B objs arrows` has a colimit. -/
+abbreviation has_wide_pushout (B : C) (objs : J → C)
+  (arrows : Π (j : J), B ⟶ objs j) : Prop :=
+has_colimit (wide_pushout_shape.wide_span B objs arrows)
+
+/-- A choice of wide pullback. -/
+noncomputable
+abbreviation wide_pullback (B : C) (objs : J → C) (arrows : Π (j : J), objs j ⟶ B)
+  [has_wide_pullback B objs arrows] : C :=
+limit (wide_pullback_shape.wide_cospan B objs arrows)
+
+/-- A choice of wide pushout. -/
+noncomputable
+abbreviation wide_pushout (B : C) (objs : J → C) (arrows : Π (j : J), B ⟶ objs j)
+  [has_wide_pushout B objs arrows] : C :=
+colimit (wide_pushout_shape.wide_span B objs arrows)
+
+variable (C)
+
+namespace wide_pullback
+
+variables {C} {B : C} {objs : J → C} {arrows : Π (j : J), objs j ⟶ B}
+variables [has_wide_pullback B objs arrows]
+
+/-- The `j`-th projection from the pullback. -/
+noncomputable
+abbreviation π (j : J) : wide_pullback _ _ arrows ⟶ objs j :=
+limit.π (wide_pullback_shape.wide_cospan _ _ _) (option.some j)
+
+/-- The unique map to the base from the pullback. -/
+noncomputable
+abbreviation base : wide_pullback _ _ arrows ⟶ B :=
+limit.π (wide_pullback_shape.wide_cospan _ _ _) option.none
+
+@[simp]
+lemma π_arrow (j : J) : (π j : wide_pullback _ _ arrows ⟶ _) ≫ arrows _ = base :=
+by apply (limit.cone (wide_pullback_shape.wide_cospan B objs arrows)).w
+  (wide_pullback_shape.hom.term j)
+
+/-- Lift a a collection of morphisms to a morphism to the pullback. -/
```

Comment:
I don't know if this one was fixed in the dependent PR. Otherwise it should be fixed here. (No need to fix it in the other PR, I think.)
```suggestion
/-- Lift a collection of morphisms to a morphism to the pullback. -/
```

Reply:
Thanks. It's now fixed.